docs/release.md

```diff
@@ -0,0 +1,187 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Release Team](#release-team)
+- [Process](#process)
+  - [Prerelease](#prerelease)
+  - [Stable release](#stable-release)
+  - [Post release](#post-release)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The Design System follows a time-based release model where we deliver a stable
+`minor` update every two weeks. The full schedule for releases is available
+[here](https://github.com/carbon-design-system/carbon/wiki/Release-radar).
+
+We also publish prereleases before every `minor` release. This prerelease
+happens several days before the stable release. This offers an integration
+window where the prerelease can be tested on products before the stable release.
+
+We also ship security and bug fixes in `patch` releases. This will be shipped
+as-needed and do not follow a specific schedule.
+
+## Release Team
+
+The release team is responsible for coordinating the `minor` and `patch`
+releases for the Design System in a given week. This group is composed of a
+release lead and sidekick. The release lead is responsible for:
+
+- Managing the release itself, including
+  - Testing
+  - Publishing
+  - Support
+- Helping the release sidekick understand and run through the release process,
+  where appropriate
+
+The release sidekick is responsible for:
+
+- Learning how to run the release process if this is your first time on the
+  release team
+- Helping out the release lead in the release process, this includes helping
+  with testing, publishing, support, and more
+
+## Process
+
+When going through a release, the release team will go through the following
+checkpoints:
+
+| Checkpoint                       | Description                                                                                                              |
+| :------------------------------- | :----------------------------------------------------------------------------------------------------------------------- |
+| [Prerelease](#prerelease)        | Publish a prerelease that will be used to test out the release candidate before becoming stabilized                      |
+| [Stable release](#stablerelease) | Graduate the prerelease into a stable release that is available through packages on NPM                                  |
+| [Post release](#postrelease)     | Support the latest stable release and address any issues that may come up as a result of promoting the release to stable |
+
+### Prerelease
+
+The prerelease occurs on the first Tuesday of a sprint. During this stage, the
+release team will need to do the following:
+
+- [ ] Run the
+      [Version Workflow](https://github.com/carbon-design-system/carbon/actions/workflows/version.yml)
+      to automatically generate the prerelease versions for packages
```

Comment:
This is all great! The only thing I think could maybe use more clarification is this step. What does running the Version Workflow mean or look like? This is the automated part, right? I just wouldn't know what this entails from just reading it.

Reply:
@jnm2377 great question, when you visit the link it should take you to this page where you can click "run workflow"

![image](https://user-images.githubusercontent.com/3901764/169147024-32b18e39-ab9a-4048-a3f6-75d4ab94d37e.png)

Reply:
Ah, this is great. Could we add this screenshot to the docs as an example?